picamera2/encoders/h264_encoder.py

```diff
@@ -157,9 +170,9 @@ def thread_poll(self, buf_available):
                         # Write output to file
                         b = self.bufs[buf.index][0].read(buf.m.planes[0].bytesused)
```

Comment:
At some point we should probably experiment with mmaping the DMABUFs that we get from the encoder. The function that gets this buffer (the outputframe method) would be responsible for copying it if they need to. But again, that's for a subsequent PR, not this one!

Reply:
Yep, that makes sense, mentioned that in the h264 encoder issue I created, so as not to forget